src/plugins/d3/renderer/utils/symbol.ts

```diff
@@ -0,0 +1,38 @@
+import {symbolDiamond2, symbolCircle, symbolSquare, symbolTriangle2} from 'd3';
+
+import {DotStyle} from '../../../../constants';
+
+export const getScatterStyle = (index: number) => {
```

Comment:
If this is a utility only for scatter, then it's better to put it next to it. If not, then it's worth renaming the method somehow (without the **scatter** in the name)